### PR TITLE
Add codox

### PR DESCRIPTION
**Pull request by andrewberls**

### CODE REVIEW

File: doc/clj-btable.core.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```
Comment (elliot42) [inline doc/clj-btable.core.html]: Was this intentional?

**Comment by elliot42** [comment]
It looks like a bunch of generated stuff got checked in?

**Comment by andrewberls** [comment]
Yes that's on purpose, those are the codox files which we've been keeping in vcs http://framed-data.github.io/clj-btable

**Comment by elliot42** [comment]
:ok:

**Pull request merged**